src/libraries/Microsoft.PowerFx.Core/Public/Config/NameSymbol.cs

```diff
@@ -24,14 +24,14 @@ namespace Microsoft.PowerFx
     [DebuggerDisplay("{Name} ({Owner.DebugName}:{SlotIndex})")]
     internal class NameSymbol : ISymbolSlot
     {
-        public NameSymbol(string name, bool mutable)
+        public NameSymbol(string name, SymbolProperties props)
         {
             Name = name;
```

Comment:
>name [](http://example.com/codeflow?start=19&length=4)

Could add  a check on name as it can't be null

Reply:
this is internal, caller has it.